subprojects/docs/README.md

```diff
@@ -64,59 +64,65 @@ You will find these references useful when authoring AsciiDoc:
  - [Asciidoctor User Manual](https://asciidoctor.org/docs/user-manual/)
  - [Asciidoctor Gradle Plugin Reference](https://asciidoctor.org/docs/asciidoctor-gradle-plugin/)
- 
-### Linking to References
 
+### Using Asciidoctor
 
+To write a chapter in Asciidoctor format, simply place it in `src/docs/userguide` called `<chapter>.adoc`.
 
 ### Code Samples
 
 Samples and output belong under `src/samples` and are typically included in the user manual. This is a typical example:
 
-```asciidoc
-[source.multi-language-sample,groovy]             
-.init.gradle
-----
-include::{samples-path}/userguide/initScripts/customLogger/init.gradle[]
-----
-[source.multi-language-sample,kotlin]             
-.init.gradle.kts
-----
-include::{samples-path}/userguide/initScripts/customLoggerKts/init.gradle.kts[]
-----
+#### Example multi-language sample file listing
+This shows Groovy and Kotlin sample projects under "sample-dir" which is defined as "$projectDir/src/samples".
 
-.Output of **`gradle -I init.gradle build`**
-----
-> gradle -I init.gradle build
-include::{samples-path}/userguide/initScripts/customLogger/custom_logging_ui.out[]
-----
+```
+subprojects/docs/src/samples/
+└── userguide/initScripts/customLogger/
+    ├── customLogger.out
+    ├── customLogger.sample.conf
+    ├── groovy
+    │   ├── build.gradle
+    │   ├── init.gradle
+    │   └── settings.gradle
+    └── kotlin
+        ├── build.gradle.kts
+        ├── customLogger.init.gradle.kts
+        └── settings.gradle.kts
 ```
 
-Notice that the Groovy and Kotlin samples are declared with `[source.multi-language-sample,<language>]`. 
-This notes to our front-end code to group these 2 samples and show them with selector tabs. 
-We intend to implement embedded samples so that the sample sources are in the user manual itself. 
+Note here that there are 2 sample projects under `userguide/initScripts/customLogger/`: one for the Groovy DSL and one for Kotlin DSL. Also note that there is only 1 `customLogger.sample.conf` file that tells Exemplar how to execute both groovy and kotlin samples, with 1 `customLogger.out` file proving the output is identical between the two.
 
-### Using Asciidoctor
+#### Example Asciidoctor multi-language sample declaration
 
-To write a chapter in Asciidoctor format, simply place it in `src/docs/userguide` called `<chapter>.adoc`.
-=======
 ```asciidoc
-// tag::something[]
-interesting code
-// end::something[]
+.Customizing what Gradle logs
+====
+include::sample[dir="userguide/initScripts/customLogger/groovy",files="init.gradle[]"]
 
-some other code
-```
+include::sample[dir="userguide/initScripts/customLogger/kotlin",files="customLogger.init.gradle.kts[]"]
+====
 
-They should be included using the `tag` or `tags` attribute this way:
-
-```asciidoc
-.build.gradle
+[.multi-language-text.lang-groovy]
 ----
-include::{samples-path}/foo/build.gradle[tag=something]
+$ gradle -I init.gradle build
```

Comment:
❓ the Groovy init script could be renamed `customLogger.init.gradle` or `customLogger.gradle`